packages/assets-controllers/src/TokensController.ts

```diff
@@ -273,29 +274,32 @@ export class TokensController extends BaseController<
     this.#abortController.abort();
     this.#abortController = new AbortController();
     this.#chainId = chainId;
+    const selectedAccount = this.messagingSystem.call(
+      'AccountsController:getAccount',
+      this.#selectedAccountId,
+    );
```

Comment:
```suggestion
    const selectedAddress = this.#getSelectedAddress();
```

And use `selectedAddress` afterward

Reply:
WDYT about this suggestion ⬆️?